content/docs/dispensary/patients.md

```diff
@@ -14,33 +14,36 @@ toc = true
 top = false
 +++
 
-Patients can be added and their information edited from this page. 
+Patients can be added and their information edited from this page.
 
 ### Patients
 
 To use patients, ensure that your store is set to `Dispensary` mode (see the [virtual stores](https://docs.msupply.org.nz/other_stuff:virtual_stores#store_type) documentation for how to do this).
 
 ### Open the Patient Menu
 
-Click on `Dispensary` > `Patients` in the navigation panel. 
+Click on `Dispensary` > `Patients` in the navigation panel.
 
 ![dispensary menu](/docs/dispensary/images/dispensary_menu.png)
 
 You will be presented with a list of patients visible to your store.
 
 ### Patient List
 
-The list of patients is divided into 8 columns: 
+![](/docs/dispensary/images/patient-list-view.png)
 
-| Column      | Description                         |
-| :---------- | :---------------------------------- |
-| **Patient ID**    | The ID of the patient                |
-| **NUIC**  | National unique identifier code  |
-| **First name**  | First name  |
-| **Last name**  | Last name  |
-| **Gender** | Gender | 
-| **Date of Birth**  | The patient's date of birth  |
-| **Deceased** | Whether the patient is deceased  |
+The list of patients is divided into 8 columns:
+
+| Column            | Description                      |
+| :---------------- | :------------------------------- |
+| **Patient ID**    | The ID of the patient            |
+| **NUIC**          | National unique identifier code  |
+| **Created**       | The date the patient was created |
+| **First name**    | First name                       |
+| **Last name**     | Last name                        |
+| **Gender**        | Gender                           |
+| **Date of Birth** | The patient's date of birth      |
+| **Deceased**      | Whether the patient is deceased  |
 
 2. The list can display a fixed number of patients per page. On the bottom left corner, you can see how many patients are currently displayed on your screen.
```

Comment:
Not related to your changes, but why are these next bits numbered... from 2??

Reply:
Cause missed the 1 above XD